aspnet/mvc/models/validation.rst

```diff
@@ -41,172 +41,155 @@ Simply reading through the model reveals the rules about data for this app, maki
 
-  <script src="~/lib/jquery/jquery.js"></script>
-  <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
-  <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+You must have a view with the proper JavaScript script references in place for client side validation to work as you see here.
 
-MVC uses validation attributes in addition to type metadata from model properties to validate data and display any error messages using JavaScript. When you use MVC to render form elements from a model using `Tag Helpers <https://docs.asp.net/en/latest/mvc/views/tag-helpers/index.html>`_ or `HTML helpers <https://docs.asp.net/en/latest/mvc/views/html-helpers.html>`_ it will add HTML 5 `data- attributes <http://w3c.github.io/html/dom.html#embedding-custom-non-visible-data-with-the-data-attributes>`_ in the form elements that need validation, as shown below. MVC generates the ``data-`` attributes for both built-in and custom attributes. The ``data-val-required`` attribute below contains an error message to display if the user doesn't fill in the release date field, and that message displays in the accompanying ``<span>`` element. You can display validation errors on the client using the relevant tag helpers as shown here:
+.. literalinclude:: validation/sample/Views/Shared/_Layout.cshtml
+   :language: html
+   :lines: 37
+   :dedent: 4
+.. literalinclude:: validation/sample/views/Shared/_ValidationScriptsPartial.cshtml
+   :language: html
 
-.. code-block:: html
- :emphasize-lines: 7, 8
+MVC uses validation attributes in addition to type metadata from model properties to validate data and display any error messages using JavaScript. When you use MVC to render form elements from a model using `Tag Helpers <https://docs.asp.net/en/latest/mvc/views/tag-helpers/index.html>`_ or `HTML helpers <https://docs.asp.net/en/latest/mvc/views/html-helpers.html>`_ it will add HTML 5 `data- attributes <http://w3c.github.io/html/dom.html#embedding-custom-non-visible-data-with-the-data-attributes>`_ in the form elements that need validation, as shown below. MVC generates the ``data-`` attributes for both built-in and custom attributes. You can display validation errors on the client using the relevant tag helpers as shown here:
 
-  <form asp-action="Create">
-    <div class="form-horizontal">
-      <h4>Movie</h4>
-        <div class="form-group">
-          <label asp-for="ReleaseDate" class="col-md-2 control-label"></label>
-          <div class="col-md-10">
-            <input asp-for="ReleaseDate" class="form-control" />
-            <span asp-validation-for="ReleaseDate" class="text-danger"></span>
-          </div>		
-      </div>	
-    </div>  
-  </form>
+.. literalinclude:: validation/sample/Views/Movies/Create.cshtml
+   :language: html
+   :lines: 19-25
+   :dedent: 8
+   :emphasize-lines: 22-23
```

Comment:
@dougbu The emphasize line numbers are actually relative to the included text. So they need to be between 1 and 7 in this case.